core/src/test/java/org/apache/calcite/test/SqlJsonFunctionsTest.java

```diff
@@ -521,13 +521,48 @@
 
   @Test public void testJsonRemove() {
     assertJsonRemove(
-        JsonFunctions.jsonValueExpression("{\"a\": 1, \"b\": [2]}"),
-        new String[]{"$.a"},
-        is("{\"b\":[2]}"));
+            JsonFunctions.jsonValueExpression("{\"a\": 1, \"b\": [2]}"),
```

Comment:
Please make sure you use the right indent: the continuous indention is 4 in Calcite code. So do codes in other new added methods.